pkg/providers/aws/cluster_network_provider.go

```diff
@@ -685,6 +686,63 @@ func (n *NetworkProvider) IsEnabled(ctx context.Context) (bool, error) {
 	return len(validBundledVPCSubnets) == 0, nil
 }
 
+// DeleteBundledCloudResources returns an error on any error deleting of the following resources
```

Comment:
I put this function in cluster_network_provider.go because it has access to the three required sessions for deleting the related resources. Initially I was putting this in cluster_vpc.go. I can still move it back there, but it will require some additional code to get the required aws sessions in place to do it all in one go. I've left a note here that it relates to the bundled networking unlike the other code in this file. It might suit. If not just let me know. :)

Reply:
I think this is fine 👍